CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## 1.5.3 2018-12-20
+
+Linkerd 1.5.3 includes a Finagle upgrade that fixes some memory related issues and adds support for 
+more efficient HTTP/1.1 routing for large HTTP requests. This release also improves 
+Linkerd's start up script to can run with newer versions of Java. Finally, this release adds a
+new feature for gRPC response classifiers that allow for added flexibility in classifying user 
```

Comment:
Simpler: Finally, this release adds a new gRPC response classifier that may be configured with gRPC status codes.